samples/csharp_dotnetcore/13.core-bot/CoreBot.csproj

```diff
@@ -5,14 +5,6 @@
     <LangVersion>latest</LangVersion>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Content Remove="Cards\welcomeCard.json" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <EmbeddedResource Include="Cards\welcomeCard.json" />
-  </ItemGroup>
-
```

Comment:
This will break the sample. The Cards folder is not published because it is consumed as an embedded resource. See here: https://github.com/microsoft/BotBuilder-Samples/blob/648a181d96184ca84204b1d82fa7ecfddc5adb19/samples/csharp_dotnetcore/13.core-bot/Bots/DialogAndWelcomeBot.cs#L45